src/markdown/lists.scss

```
@@ -26,6 +26,12 @@
     list-style-type: lower-roman;
   }
 
+  // Reset <ol> style to decimal (HTML default) specifically for AsciiDoc
+  // <div><ol> construction (doesn't affect MarkDown)
+  div ol:not([type]) {
```

Yeah, it seems the `<div>` as a wrapper is the only thing that differentiates AsciiDoc and MarkDown. Should we use the `>` child selector to make it even more specific?

```suggestion
  div > ol:not([type]) {
```

---

Definitely! I missed that.
Thanks

---

Fixed-up the commit with child selector `div > ol`